search-translations.js

```diff
@@ -0,0 +1,38 @@
+//requiring path and fs modules
+const path = require('path');
+const fs = require('fs');
+
+console.log('Running search-translations...');
+
+const directoryPath = path.join(__dirname, process.argv[2]);
+const allFiles = walk(directoryPath);
```

Comment:
exclude well known folders or folders mentioned in well known locations `.gitignore` etc

Reply:
`readdirSync` already ignores it that's why I didn't add it